### ISSUE TITLE
User having problems calling Jenkins Linter 

### ISSUE THREAD

**Issue by de-jcup** [de-jcup/eclipse-jenkins-editor#81]
This is a migrated issue from https://marketplace.eclipse.org/content/jenkins-editor#comment-7424 :

> I tried the jenkins linter several times but all I get is 'Jenkins CLI call failed' so my guess it, the plugin forgets to store the credentials from the settings dialog somewhere.
> 
> First some information about my setup: I am running a jenkins-server 2.249.2 on a server on my network. In my user settings on jenkins I created an API token. Then I switched to my eclipse (9090-09 with jenkins editor 1.7.0), opened the settings of jenkins editor, checked the jenkins URL, added my user name in 'credentials' and added my API token. Once I pressed 'Test connection' and got the message 'Connection test successful'.
> 
> Whatever I do next: press 'apply' first or directly 'apply and close', whenever I try to lint a jenkinsfile all I get is 'Jenkins CLI call failed' with the following message in the details view:
> 
> "JenkinsLinterCLIResult:
> - exitCode:-2 (Linter was not executed at all)
> - failureMessage:Access to Jenkins was not possible by command:
> 
> java -jar C:\Apps\jenkins-cli\jenkins-cli.jar -s %URL_NOT_ALLOWED_HERE% -http -noCertificateCheck -auth null:******* declarative-linter
> 
> Maybe credentials not valid or hostname/firewall problems.
> Please check Jenkins CLI setup in preferences
> -Output was:
> 
> "
> 
>  
> 
> BTW: when I copied the command line java -jar blabla to a cmd window and replaced the pair 'null:****' with my actual user/API-token and pasted a jenkinsfile to the console I got: 'Jenkinsfile successfully validated.'
> 
> So my idea about lost settings seems quite valid.
> 
> Could you please fix this?
> 
> Best regards,
> Christoph

**Comment by de-jcup** [comment]
I mentioned link to https://github.com/de-jcup/eclipse-jenkins-editor/wiki/Setup at the marketplace. Maybe this has fixed the problem, because
```
java -jar C:\Apps\jenkins-cli\jenkins-cli.jar -s %URL_NOT_ALLOWED_HERE% -http -noCertificateCheck -auth null:******* declarative-linter
```
says the user name is "null" which means no user credentials set in eclipse secure store.

**Comment by chriskloe** [comment]
Hi Albert,

sorry, I did not notice your reply for a while. Please have a look at issue #77 I created at the same time as the marketplace post.
There as well as in the original post I stated, I actually set the credentials in the settings but in my opinion they are not stored.

**Comment by culmat** [comment]
I think this can be closed as duplicate of #77
